src/tests/homework.spec.js

```diff
@@ -0,0 +1,124 @@
+import { expect, test } from "@playwright/test";
+
+import {
+  ApplicationTexts,
+  invalidPassword,
+  password,
+  userFullName,
+} from "../fixtures/fixtures.js";
+
+import { LoginPage } from "./pages/login.page.js";
+import { RegistrationPage } from "./pages/registration.page.js";
+
+test.describe("Registration Page is Correctly Displayed", async () => {
+  test.beforeEach(async ({ page }) => {
+    const loginPage = new LoginPage(page);
```

Comment:
Pro účely tohoto zadání není potřeba více page object modelů než jeden, a to RegistrationPage. Do něj se dá vložit metoda pro otevření stránky `open()`